### PR TITLE
Bug 2103554: Bump Jenkins 2.346.1 and various plugins bumps [release-4.8]

### PR DESCRIPTION
**Pull request by akram**
Bump Jenkins 2.346.1 and various plugins bumps

### CODE REVIEW

File: 2/contrib/openshift/base-plugins.txt [hunk 1]
```diff
@@ -14,7 +14,7 @@ groovy:2.4
 htmlpublisher:1.25
 jira:3.7.1
 job-dsl:1.77
-junit:1.52
+junit:1119.1121.vc43d0fc45561
```
Comment (coreydaley) [inline 2/contrib/openshift/base-plugins.txt:17]: Why do we end up going back and forth between versions such as `1.52` and `1119.xxxx.xxxx.xxxx`?

Comment (akram) [reply 2/contrib/openshift/base-plugins.txt:17]: This version was required by another plugin (workflow-api:1144.v61c3180fa_03f iirc)

**Comment by maximili0n** [comment]
Hi @akram @coreydaley ,
is this going to be released soon?

Best regards
Maximilian Conrad

**Comment by coreydaley** [comment]
We are currently working on it.

**Pull request closed without merging**